Dockerfile

```diff
@@ -52,9 +52,8 @@ COPY entrypoint.sh /entrypoint.sh
 #   3000 – service port, for client connections
 #   3001 – fabric port, for cluster communication
 #   3002 – mesh port, for cluster heartbeat
-#   3003 – info port
 #
-EXPOSE 3000 3001 3002 3003
+EXPOSE 3000 3001 3002
```

Comment:
@volmarl I want to remove 3003, so I did it here and in the examples. Anything else?

Reply:
That should be it  in terms of expose ports.